src/System.ServiceModel.Security/tests/ServiceModel/SecurityAlgorithmSuiteTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using Infrastructure.Common;
+using System;
+using System.Reflection;
+using System.ServiceModel;
+using System.ServiceModel.Security;
+using Xunit;
+
+public static class SecurityAlgorithmSuiteTest
+{
+    [WcfTheory]
+    [InlineData(nameof(System.ServiceModel.Security.TripleDesSecurityAlgorithmSuite.DefaultCanonicalizationAlgorithm), "http://www.w3.org/2001/10/xml-exc-c14n#")]
+    [InlineData(nameof(System.ServiceModel.Security.TripleDesSecurityAlgorithmSuite.DefaultDigestAlgorithm), "http://www.w3.org/2000/09/xmldsig#sha1")]
```

Comment:
Can you simplify the naming to remove the S.SM.Security namespace as you are using that namespace already?

Reply:
Done